# Patch to fix hanging ipfunc test on sparc. Not for upstream.

--- ipython-0.10/IPython/testing/plugin/dtexample.py	2009-04-15 17:58:05.000000000 -0700
+++ ipython-0.10/IPython/testing/plugin/dtexample.py	2015-07-07 09:41:08.098281161 -0700
@@ -23,7 +23,7 @@
     """
     return 'pyfunc'
 
-
+'''
 def ipfunc():
     """Some ipython tests...
 
@@ -71,7 +71,7 @@
     Out[9]: 'ipfunc'
     """
     return 'ipfunc'
-
+'''
 
 def ranfunc():
     """A function with some random output.
